packages/nimble-components/src/rich-text/base/index.ts

```diff
@@ -21,7 +21,7 @@ export abstract class RichText extends FoundationElement {
     public readonly childItems: Element[] = [];
 
     @observable
-    protected configuration?: Configuration;
+    public configuration?: Configuration;
```

Comment:
Mark internal

Reply:
With the latest changes doesn't seem to need to be public anymore?

Reply:
Current test cases are using this configuration. We will revisit the test scripts and update this.

Reply:
Updated it to `protected`. Generated the required configuration dynamically based on mappings just to parse and get the HTML in page object `getParserMentionConfigForUser()`, replaced the actual element configuration usage.